src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -275,7 +275,7 @@ private void abortRunningJobsThatMatch(@Nonnull StashCause stashCause) {
     }
 
     return ParameterizedJobMixIn.scheduleBuild2(
-        job, -1, new CauseAction(cause), new ParametersAction(values));
+        job, -1, new CauseAction(cause), new StashQueueAction(), new ParametersAction(values));
```

Comment:
minor: we might make it a singleton if it's stateless

Reply:
This class won't be stateless if we start using it as a replacement for `cancelPreviousJobsInQueueThatMatch`.